### PR TITLE
Harsh/add all data in duplicate event

### LINKED ISSUE #5871: [CAL-532] Duplicating Event Type Issues
When duplicating an existing event type, most of the settings of the existing event type are not applied to the new event type. 

The ideal scenario is for a 1:1 duplication of the settings in the event type in it's entirety.

<sub>[CAL-532](https://linear.app/calcom/issue/CAL-532/duplicating-event-type-issues)</sub>

### PR DESCRIPTION
**Pull request by harshsinghatz**
## What does this PR do?
- while duplicating an only all the data is not being duplicated
- This PR also adds from the data to the duplicated event.

Fixes # (issue)
- [reference issue](https://github.com/calcom/cal.com/issues/5835)

[Loom video](https://www.loom.com/share/c5eb1f6b3f4743d1aa3d050d2d18dae9)

**Environment**: Staging(main branch) / Production

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## How should this be tested?

- [ ] Try duplicating an event alot of configuration changes. eg: buffer limit,etc and check if they're duplicated or not.

### CODE REVIEW

**Comment by harshsinghatz** [comment]
@PeerRich @alannnc @hariombalhara @CarinaWolli  can you review this and suggest the changes for the comments above? (bit of a time crunch that is why not able to give more time on this issue)

**Review by alannnc — COMMENTED** [review]
- When duplicating, modal should put old event type name with "Event Name" Copy or "Event Name" (1).
- When duplicating event type this message is received: `Argument bookingLimits for data.bookingLimits must not be null. Please use undefined instead.`

File: packages/prisma/schema.prisma [hunk 1]
```diff
@@ -84,6 +84,7 @@ model EventType {
   workflows               WorkflowsOnEventTypes[]
   /// @zod.custom(imports.bookingLimitsType)
   bookingLimits           Json?
+  duplicatedFromEventId   Int?
```
Comment (alannnc) [inline packages/prisma/schema.prisma:87]: I think we don't need migrations for saving this.

**Comment by alannnc** [comment]
Thanks for your contribution! Closed in favor of #5888

**Pull request closed without merging**